kbase-extension/static/kbase/js/widgets/appWidgets2/view/floatView.js

```diff
@@ -1,13 +1,10 @@
-define([
-    'bluebird',
-    'base/js/namespace',
-    'kb_common/html',
-    'common/events',
-    'common/ui',
-    'common/props',
-
-    'bootstrap',
-], (Promise, Jupyter, html, Events, UI, Props) => {
+define(['bluebird', 'common/html', 'common/ui', 'common/props', '../inputUtils', 'bootstrap'], (
```

Comment:
Like `intView`, this doesn't actually validate anything. So all changes are linting and reorganizing.